src/main/java/com/google/cloud/genomics/dataflow/utils/GenomicsDatasetOptions.java

```diff
@@ -13,22 +13,21 @@
  */
 package com.google.cloud.genomics.dataflow.utils;
 
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.List;
+import java.util.logging.Logger;
+
 import com.google.api.services.genomics.Genomics;
 import com.google.api.services.genomics.model.SearchVariantsRequest;
 import com.google.cloud.dataflow.sdk.options.Default;
 import com.google.cloud.dataflow.sdk.options.Description;
-import com.google.cloud.dataflow.sdk.options.Validation.Required;
 import com.google.cloud.genomics.utils.Contig;
 import com.google.cloud.genomics.utils.Contig.SexChromosomeFilter;
 import com.google.cloud.genomics.utils.GenomicsFactory;
 import com.google.common.base.Preconditions;
 import com.google.common.collect.Lists;
 
-import java.io.IOException;
-import java.security.GeneralSecurityException;
-import java.util.List;
-import java.util.logging.Logger;
-
 /**
  * A common options class for all pipelines that operate over a single dataset and write their
  * analysis to a file.
```

Comment:
If this is now going to be for pipelines that may write their output to a file, we should update the comment. Alternatively, we could split into multiple option classes: one for setting the single input, one for setting a single-file output, and one for setting a bigquery output. We can then combine them at will.